classes/Presenter/Store/Modules/PaypalModule.php

```diff
@@ -40,7 +40,7 @@ public function present()
         $paypalModule = [
             'paypal' => [
                 'idMerchant' => $idMerchant,
-                'paypalOnboardingLink' => false,
+                'paypalOnboardingLink' => '',
```

Comment:
What if the reason in  change the default value?

Reply:
We need a default value that is not "false". We set this value to false when we cannot retrieve an onboarding link from the api in order to display an error to the merchant.

Reply:
When false, an error message is displayed in account-list.vue